dev-bin/make-man-pages.pl

```diff
@@ -67,18 +66,20 @@ sub _make_man {
             '-M', "section:$section",
             $input,
             '-o', $output,
-        );
+        ) == 0 or die "Failed to run lowdown: $!";
     }
 }
 
 sub _make_lib_man_links {
     my $target = shift;
 
-    my $header = read_file("$Bin/../include/maxminddb.h");
+    my $header = read_file("$Bin/../include/maxminddb.h")
+        or die "Failed to read header file: $!";
     for my $proto ( $header =~ /^ *extern.+?(MMDB_\w+)\(/gsm ) {
-        open my $fh, '>', "$target/man/man3/$proto.3";
+        open my $fh, '>', "$target/man/man3/$proto.3"
+            or die "Failed to open file: $!";
         print {$fh} ".so man3/libmaxminddb.3\n";
```

Comment:
I think `print` could fail too, but it looks like `autodie` already didn't handle that!